src/middleware.js

```diff
@@ -0,0 +1,40 @@
+/*global ga*/
+
+// TODO: Fix double event on moment page load.
+
+import { RECEIVE_AUDIO } from "./actions"; 
+
+// The constant isn't exported from redux-router so I'm having to redefine it.
+// It's a bit of a hack and makes the Google Analytics code dependent on a version of redux-router.
+// An alternative way to do this is subscribe to the react router, but then you can't handle events the same way.
+const ROUTER_DID_CHANGE = "@@reduxReactRouter/routerDidChange";
+
+const gaMomentEventCategory = "Moment";
+const gaPlayEventAction = "play";
+const gaPauseEventAction = "pause";
+
+export function googleAnalytics(store) {
+  // Return a function handling actions.
+  return next => action => {
+    switch (action.type) {
+      // Send a pageview when the route changes.
+      case ROUTER_DID_CHANGE:
+        ga("set", "page", action.payload.location.pathname + action.payload.location.search);
```

Comment:
need to check if ga is defined before you use it. Also define it as a webpack external https://webpack.github.io/docs/library-and-externals.html#applications-and-externals